### PR TITLE
Use Babel retainLines option

### LINKED ISSUE #19: When running tests, errors show wrong line numbers.
The code being tested is written in ES6 modules. It seems like the jest errors that are shown are relative to the transpiled code, not the original source. How can we fix this problem? Do source maps work with babel-jest somehow?

### PR DESCRIPTION
**Pull request by fdubost**
Show right line numbers when there are errors in jest tests.
Fix #19.

### CODE REVIEW

**Comment by sebmck** [comment]
Thanks! This option hasn't been as battle tested as I'd like so I guess this is a good way of getting people to use it... The only code breaking thing it could do is cause a syntax error though.

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "babel-jest",
-  "version": "5.0.1",
+  "version": "5.1.0",
```
Comment (sebmck) [inline package.json]: Please don't bump the version. It messes things up when I do `npm release`.

**Comment by sebmck** [comment]
Released as of 5.2.0

File: package.json (hunk 1 above)
Comment (fdubost) [inline package.json]: Ok, I didn't know, sorry :blush:

**Comment by fdubost** [comment]
Thanks :smiley: !

**Comment by crubier** [comment]
I have the same problem with  babel-jest  6.0.1

**Comment by crubier** [comment]
Even when adding 

    "babel": {
        "retainLines": true
    },

To my `package.json`
... I guess the problem comes from babel though....

**Pull request merged**